back/app/services/side_fx_initiative_service.rb

```diff
@@ -45,6 +44,8 @@ def after_update(initiative, user, old_cosponsor_ids)
       LogActivityJob.perform_later(initiative, 'changed_title', user_for_activity_on_anonymizable_item(initiative, user), initiative.updated_at.to_i, payload: { change: initiative.title_multiloc_previous_change })
     end
 
+    log_activities_if_cosponsors_added(initiative, user, old_cosponsor_ids)
+
```

Comment:
Moving this to later in the method, avoids breaking the functionalities of some of the preceding actions (e.g. `remove_user_from_past_activities_with_item`)